x-pack/plugins/observability/public/hooks/use_alerts_history.ts

```diff
@@ -30,14 +31,19 @@ interface Props {
   };
 }
 interface FetchAlertsHistory {
-  totalTriggeredAlerts: number;
-  histogramTriggeredAlerts: estypes.AggregationsDateHistogramBucketKeys[];
-  error?: string;
-  avgTimeToRecoverUS: number;
+  totalTriggeredAlerts?: number;
```

Comment:
It should be optional in the first place because it depends on the API return, but the type wasn't defined. After adding the type, it's ok https://github.com/elastic/kibana/pull/157178/files/0c22ab6de46babdf992eea671ff5a9b18338f265#diff-713312266b1bc084bf30bb728994226de8feb42658e157bebe1bbd115f4fd758R45

Reply:
we are using ReactQuery now